### PR TITLE
Stablizing the industry vertical schemas

### PR DESCRIPTION
**Pull request by mjasrotia**
Please link to the issue #…

### CODE REVIEW

**Review by hanessian — APPROVED** [review]

File: components/mixins/experience-event/industry-verticals/experienceevent-flight-reservation.schema.json [hunk 1]
```diff
@@ -295,5 +295,5 @@
       "$ref": "https://ns.adobe.com/xdm/mixins/industry-verticals/experienceevent-reservation-details"
```
Comment (hanessian) [inline components/mixins/experience-event/industry-verticals/experienceevent-flight-reservation.schema.json:295]: can you please change ID here to ticket number?

File: components/mixins/experience-event/industry-verticals/experienceevent-flight-reservation.schema.json [hunk 2]
```diff
@@ -295,5 +295,5 @@
...
     }
```
Comment (hanessian) [inline components/mixins/experience-event/industry-verticals/experienceevent-flight-reservation.schema.json:296]: In this mixin, please add Aircraft Type - type of aircraft associated with the reservation 
and 
Aircraft ID - ID of the aircraft associated with the reservation.

Then in the ERD, the identity should point to Aircraft ID. NOT flight number

Comment (hanessian) [reply components/mixins/experience-event/industry-verticals/experienceevent-flight-reservation.schema.json:296]: Example value for aircraft ID can be Boeing 737

**Review by hanessian — CHANGES_REQUESTED** [review]
see comments.

**Review by hanessian — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**